common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala

```diff
@@ -18,10 +24,24 @@ object PhotographersList {
   def list(store: Store) = store.keys.toList.sortWith(_.toLowerCase < _.toLowerCase)
 
   def getPublication(store: Store, name: String): Option[String] = store.get(name)
-}
+
+  def caseInsensitiveLookup(store: Store, lookup: String) =
+    store.map { case (name, pub) =>
+      name.toLowerCase -> (name, pub)
+    }.get(lookup.toLowerCase)
```

Comment:
It could be more efficient (and readable?) to use `Map#find` instead, but there is so little data it doesn't matter that much.